src/components/table.js

```diff
@@ -0,0 +1,118 @@
+import { host, workspaceId } from "../utils/script_attributes";
+
+export class OSTable extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    const haveLoopAttribute = this.getAttribute("data-os-element") === "loop";
+    const requestHost = host || "https://app.formli.com";
+
+    if (haveLoopAttribute) {
+      const responseViewUuid = this.getAttribute("data-os-view");
+      if (responseViewUuid) {
+        const emptyElement = document.createElement("div");
+        emptyElement.innerText = "No data found";
+        emptyElement.classList.add("os-hidden");
+        emptyElement.setAttribute("data-os-element", "empty");
+        emptyElement.setAttribute("data-os-for", responseViewUuid);
+        this.appendChild(emptyElement);
+        this.dispatchLoadingEvent(true);
+
+        fetch(
+          `${requestHost}/api/v1/response_views/${responseViewUuid}/items`,
```

Comment:
@hheight I think this component knows too much.  This API endpoint is mentioned in this custom element and in the HTML form script.  Essentially both files know how a loop works.  

This custom element should use `data-os-element="loop"` but it should not implement any of the data fetching.  It should be able to get the data from `data-os-element="loop"` and manage how it is displayed on the page.

I've merged this, but this will need to be fixed in a separate PR